website/source/docs/builders/openstack.html.md

```diff
@@ -118,6 +118,9 @@ builder.
 -   `metadata` (object of key/value strings) - Glance metadata that will be
     applied to the image.
 
+-   `instance_name` (string) - Name that is applied to the server instance
+    created by Packer. If this isn't specified, the default is same as image_name.
```

Comment:
Surround _image_name_ with backticks.

Reply:
Maybe is this duplicated with previous one?